test/python/transpiler/test_inverse_cancellation.py

```diff
@@ -0,0 +1,153 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Testing InverseCancellation
+"""
+
+import numpy as np
+
+from qiskit import QuantumCircuit
+from qiskit.transpiler.exceptions import TranspilerError
+from qiskit.transpiler.passes import InverseCancellation
+from qiskit.transpiler import PassManager
+from qiskit.test import QiskitTestCase
+from qiskit.circuit.library import RXGate, HGate, CXGate, PhaseGate, XGate
+
+
+class TestInverseCancellation(QiskitTestCase):
+    def test_InverseCancellation_h(self):
+        qc = QuantumCircuit(2, 2)
```

Comment:
See if you can update the names of these tests to be a little more descriptive. Also add a docstring (documentation string) with a short description. For example:
```suggestion
    def test_basic_self_inverse(self):
        """Test that a single self-inverse gate as input can be cancelled."""
        qc = QuantumCircuit(2, 2)
```

And for the next one, it might be `test_odd_number_self_inverse` with the documentation reading something like `Test that an odd number of self-inverse gates leaves one gate remaining.`.